client/assets/components/modalContainer.scss

```diff
@@ -0,0 +1,82 @@
+.modal {
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background-color: rgba(0, 0, 0, 0.64);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  z-index: 1040;
+
+  .modal__content {
+    background-color: #fff;
+    padding: 25px 25px;
+    width: 90%;
+    max-width: 768px;
+    overflow-y: auto;
+    border-radius: rem(6px);
+
+    @media (min-width: 600px) {
```

Comment:
Can you check [this out](https://engageinteractive.co.uk/blog/em-vs-rem-vs-px) and let me know what you think about using `em` for media queries instead of `px`?

Reply:
@BolajiOlajide according to [this](https://github.com/sass-mq/sass-mq/issues/59) it's really a no issue, lemme know if i should still make the changes

Reply:
I just went through the link and I found [this link](https://stackoverflow.com/questions/29686099/media-queries-px-vs-em-vs-rem#29723215) hyperlinked to it, it might be an issue but I guess something we don't have to bother ourselves with right now (especially considering we didn't set a relative font-size on the html tag.